Automattic/jetpack
# Untangling: Update WP.com themes banner image

## Proposed changes:

Updates the image of the WP.com themes banner to display previews of actual WP.com themes.

Before | After
--- | ---
<img width="1099" alt="Screenshot 2024-04-11 at 11 22 34" src="https://github.com/Automattic/jetpack/assets/1233880/d4e60bfc-7b52-4809-9c24-8216062db0c0"> | <img width="1096" alt="Screenshot 2024-04-11 at 11 24 33" src="https://github.com/Automattic/jetpack/assets/1233880/e6015c48-3b73-462a-870a-60a3ce591f4a">



### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
- If you don't have an untangled Atomic site:
  - Create or pick a regular Atomic site
  - Go to wordpress.com/hosting-config and select your Atomic site
  - Change the admin interface to wp-admin
  - Go to `<SITE_DOMAIN>/_cli` and run this command: `wp option update wpcom_classic_early_release 1`
- Apply these changes to your untangled Atomic site:
  - Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/)
  - Go to `/wp-admin/admin.php?page=jetpack-beta&plugin=jetpack-mu-wpcom-plugin`
  - Activate this branch
  - Add the following to your `wp-config.php` file: `define( 'JETPACK_MU_WPCOM_LOAD_VIA_BETA_PLUGIN', true );`
- Go to `/wp-admin/theme-install.php`
- Make sure the new image is displayed in the WP.com themes banner
